gromacs/config.py

```diff
@@ -372,7 +317,7 @@ def _generate_template_dict(dirname):
     by external code. All template filenames are stored in
     :data:`config.templates`.
     """
-    return dict((resource_basename(fn), resource_filename(__name__, dirname+'/'+fn))
+    return dict((resource_basename(fn), resource_filename(__name__, dirname + '/' + fn))
```

Comment:
If you are correcting spaces, just correct it to use `os.path.join()` – that's what I should have used in the first place.